.travis.yml

```diff
@@ -48,6 +49,14 @@ matrix:
    env: LINKAGE=static HOMEBREW_NO_AUTO_UPDATE=1
    before_script:
     - ./configure --install-deps --disable-lz4-ext --prefix="$PWD/dest" --enable-static --disable-syslog
+ - name: "Windows MinGW-w64"
+   os: windows
+   env:
+    - SKIP_MAKE=y
+   before_install:
+    - source ./packaging/mingw-w64/travis-before-install.sh 1>&2
```

Comment:
Why is the output redirect necessary? Travis logs both stdout and stderr.

Reply:
Probably from when I was testing in docker, will get rid of it!